src/SASjs.ts

```diff
@@ -529,8 +530,8 @@ export default class SASjs {
    * Checks whether a session is active, or login is required.
    * @returns - a promise which resolves with an object containing two values - a boolean `isLoggedIn`, and a string `userName`.
    */
-  public async checkSession() {
-    return this.authManager!.checkSession()
+  public async checkSession(accessToken?: string) {
```

Comment:
is it worth making a comment that the access token is only used in SASjs server type?

Reply:
Updated TSDoc